extra/command-not-found.bash

```diff
@@ -5,6 +5,20 @@ command_not_found_handle () {
   set +o verbose
 
   mapfile -t pkgs < <(pkgfile -bv -- "$cmd" 2>/dev/null)
+
+  
+  if [[ ${#pkgs[@]} -eq 1 ]] && [[ -n "$PKGFILE_PROMPT_INSTALL_MISSING" ]]; then
+    pkg=$(echo ${pkgs[0]} | awk '{print $1}')
+    read -r -p "Install $pkg? [Y/n] " response
+    response=${response,,}
+    if [[ $response =~ ^(no|n)$ ]]; then
+      return 0 
+    fi
+    printf '\n'
+    sudo pacman -S $pkg
```

Comment:
You've already confirmed that the user wants to install, why not use --noconfirm? And, quote "$pkg" please.